static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -125,9 +125,26 @@ function SampleImagesChartPanelBody(props: {
   return (
     <ImageWrapper>
       {images.map(resource => {
+        const hasRawDomain = Boolean(resource[RAW_DOMAIN]);
+        const isRelativeUrl = resource[SPAN_DESCRIPTION].startsWith('/');
+        let src = resource[SPAN_DESCRIPTION];
+        if (isRelativeUrl && hasRawDomain) {
+          try {
+            const url = new URL(resource[SPAN_DESCRIPTION], resource[RAW_DOMAIN]);
+            src = url.href;
+          } catch {
+            Sentry.captureException(new Error('Invalid URL'), {
+              data: {
```

Comment:
There's no data field able to be provided [here](https://docs.sentry.io/platforms/javascript/enriching-events/context/#passing-context-directly), you probably want to set `tags` or `contexts` instead here.

Reply:
So strange, typescript said its available so I assumed that it would work, will update that in a PR soon!